test/helper/Puppeteer_test.js

```diff
@@ -773,6 +768,58 @@ describe('Puppeteer', function () {
       await FS.seeFile('downloads/avatar.jpg');
     });
   });
+
+  describe('#waitForClickable', () => {
+    it('should wait for clickable', async () => {
+      await I.amOnPage('/form/wait_for_clickable');
+      await I.waitForClickable({ css: 'input#text' });
+    });
+
+    it('should fail for disabled element', async () => {
+      await I.amOnPage('/form/wait_for_clickable');
+      await I.waitForClickable({ css: '#button' }, 0.1).then((isClickable) => {
+        if (isClickable) throw new Error('Element is clicable, but must be unclickable');
```

Comment:
a typo : `clicable`

Reply:
maybe we add a `const ELEMENT_IS_UNCLICKABLE_ERROR = 'Element is clicable, but must be unclickable'`

Reply:
This error is only for unit tests) and I think it is not necessary

Reply:
I see. But you don’t need to copy that text and edit them every where :P